scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -649,144 +648,138 @@ scater::plotReducedDim(merged_sce,
   guides(colour = guide_legend(override.aes = list(size = 3, alpha = 1))) +
   ggtitle("UMAP after integration with fastMNN")
 ```
-Something we can see already see from this UMAP is that the non-tumor cell types (mostly vascular endothelium, muscle cells, and monocytes) tend to cluster together and are generally far separated from the tumor cell types, which is an encouraging pattern!
-
-Let's look at a faceted version:
+This UMAP is that the normal tissue cell types (mostly vascular endothelium, muscle cells, and monocytes) tend to cluster together and are generally far separated from the tumor cell types, which is an encouraging pattern!
+Tumor cell types from different samples are all also clustering together, which is even more encouraging that we had successful integration.
 
+However, it's a bit challenging to see all the points given the amount of overlap in the plot.
+One way we can see all the points a bit better is to facet the plot by sample, using `facet_wrap()` from the `ggplot2` package (which we can do because `scater::plotReducedDim()` returns a `ggplot2` object):
 
-```{r fastmnn umap celltypes faceted}
+```{r fastmnn umap celltype faceted}
 scater::plotReducedDim(merged_sce,
                        dimred = "fastMNN_UMAP",
-                       # color by broad celltypes
                        colour_by = "celltype_broad",
                        point_size = 0.5,
-                       point_alpha = 0.2, 
-                       other_fields = "library") +
+                       point_alpha = 0.2,
+                       # Allow for faceting by a variable using `other_fields`:
+                       other_fields = "celltype_broad") +
   guides(colour = guide_legend(override.aes = list(size = 3, alpha = 1))) +
   ggtitle("UMAP after integration with fastMNN") +
-  facet_wrap(~library)
+  # Facet by celltype_broad
+  facet_wrap(vars(celltype_broad)) +
+  # Use a theme with background grid to more easily compare panel coordinates
+  theme_bw() 
 ```
 
-Now is a good time to remember we actually have two cell type variables: `celltype_broad` and `celltype_fine`, where the latter contains finer-grained Tumor characterizations based on differentiation stage.
+All `Tumor` cell types appear to group with one another, with the main exception of `Tumor_Mesoderm` (along with what looks like one `Tumor_Myoblast` cell) for which a small number of cells are located with healthy muscle tissue.
+This _potentially_ suggests that these Tumor cells were incorrectly classified and are actually not cancerous, which is a hypothesis one could further explore, but on the other hand RMS does share many features with muscle tissue.
```

Comment:
I think I would rephrase this and instead of saying that the tumor cells were maybe misclassified, say that there are multiple options for this to happen such as misclassification, cells share similar patterns with another cell type. That way you present them as both equally possible and aren't really giving an opinion either way since we aren't disease experts.

Reply:
I might also just leave this out? We can let people make their own hypotheses!